tests/expr_and_series/reduction_test.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pytest
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+
+@pytest.mark.parametrize(
+    ("expr", "expected"),
+    [
+        (
+            [nw.col("a").min().alias("min"), nw.col("a", "b").mean()],
+            {"min": [1], "a": [2], "b": [5]},
+        ),
+        ([(nw.col("a") + nw.col("b").max()).alias("x")], {"x": [7, 8, 9]}),
+        ([nw.col("a"), nw.col("b").min()], {"a": [1, 2, 3], "b": [4, 4, 4]}),
+        ([nw.col("a").max(), nw.col("b")], {"a": [3, 3, 3], "b": [4, 5, 6]}),
+        (
+            [nw.col("a"), nw.col("b").min().alias("min")],
+            {"a": [1, 2, 3], "min": [4, 4, 4]},
+        ),
+    ],
+    ids=range(5),
+)
+def test_scalar_reduction_select(
+    request: Any, constructor: Any, expr: list[Any], expected: dict[str, list[Any]]
+) -> None:
+    if "dask" in str(constructor) and int(request.node.callspec.id[-1]) != 1:
```

Comment:
Don't panic (just yet): this friendly `request.node.callspec.id` will look like the following: `<constructor_name>-<id>`. The `id` is specified in `@pytest.mark.parametrize` and ranges from 0 to 4 (number of test cases).

As dask passes the second test which corresponds to `request.node.callspec.id="dask_lazy_constructor-1"`, I am extracting the id value



polars_lazy_constructor-4